### ISSUE TITLE
Unable to knit multiple frormats from menu bar

### ISSUE THREAD

**Issue by jrminter** [rstudio/rstudio#6538]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.929
    OS Version : Win 10 and MacOS  
    R Version       : 3.6.2

### Steps to reproduce the problem
Try to add the YAML from the pull-down location next to the "knit" button to add different output formats.

### Describe the problem in detail

This has worked for years and was one of the most helpful features in RStudio

### Describe the behavior you expected
I expected to be able to automatically add the correct YAML

- [ X] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports .  I did not find a previous bug report.
- [ X] I have installed the latest version of RStudio and confirmed that the issue still persists. I alsoe tried 2 earlier versions where I had installers
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [ x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

I do thank you for making this available

**Comment by kevinushey** [comment]
Thanks for the bug report. Unfortunately, I'm not able to reproduce. I think we'll need some more details and, if at all possible, a reproducible example.

**Comment by rschauner** [comment]
I have been having problems using the knit button for even the default yaml setting. When I click the "knit" button, the Rmd starts rendering and suddenly stops with no error (or output). However, running `rmarkdown::render()` successfully completes the render. I made a minimal reproducible example here. And, if I took over this thread, I apologize in advance. 

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = T)
library(Seurat)
```

```{r}
x <- 1

x
```
